packages/devtools/src/lib/devtools.spec.ts

```diff
@@ -15,9 +15,13 @@ describe('devtools', () => {
         connect: jest.fn().mockImplementation(() => instanceMock),
       },
     },
+    configurable: true,
+    writable: true,
   });
 
-  createEntitiesStore('books');
```

Comment:
I need to clear the store or I can not add a second `describe` as the store is in an unknown state it makes the state harder to assert.